frontend/src/components/Farm.js

```diff
@@ -42,24 +47,49 @@ const Farm = ({ farm }) => {
     micropython_array.push(<Snek size="micropython" length={5} up={i%2==0 ? state.up : !state.up}/>)
   }
 
+  const Micropythons = () => {
+    let micropys = [];
+    for (var i = 0; i < micropythons; i++) {
+      micropys.push(<Snek key={i} size="medium" length={5} up={state.up}/>)
+    }
+    return(
+      <div className="micropythons">
+        {micropys}
+      </div>
+    )
+  }
+
   if(state.curlyboi_snektions > 0) {
```

Comment:
This `if` is old code too that's duplicated inside "curlyboi"

Reply:
Thanks for spotting that -- i'm surprised i didn't see it before D: